### ISSUE TITLE
Does Docus Cards have option for links?

### ISSUE THREAD

**Issue by otabekoff** [nuxt-themes/docus#778]
![image](https://user-images.githubusercontent.com/60849894/213884066-194195b6-cda6-4b10-87c8-1feb64b65cdd.png)

I mean that is it possible to use the cards as links? We click onto them and they bring into other pages.

**Comment by bdrtsky** [comment]
Why not just use a link in `title`?

There's no straightforward way to do what you ask, because `description` could also contain links, and you can't have links inside links; and if doing navigation programmatically there's no simple way to catch clicks on links inside card while also differing if programmatic link is internal or external.

**Comment by Atinux** [comment]
I agree with @bdrtsky about adding the link on the title, learn more on https://css-tricks.com/block-links-are-a-pain-and-maybe-just-a-bad-idea/

**Comment by lucpotage** [comment]
I was in the same situation and wanted to add a link to my card.

-  _A community of Web builders_ section in nuxt.com homepage, each card is clickable, which is very affordant.
- The Docs index page with shortcuts to subpages. I find it great to have tiles that are clickable.

I get the point that a title could have a link, but it’s up to the author to properly use the component.

I would like to propose something but I don’t see any Card component in this repo. Where are they stored?

**Comment by Tahul** [comment]
```md
::nuxt-link{to="your-link"}
   ::card
   Your Card Content
   ::
::
```

I think with that snippet, you can easily make the whole card a link.

**Comment by Tahul** [comment]
I mean it's up to you how you use the semantics, but if you wrap the whole card in a link then there's no point in adding a link in the middle of its content.

**Comment by lucpotage** [comment]
Yes, your approach is best @Tahul. Unfortunately, I could not make it work.
Here is a playground where wrapping a `card` with `nuxt-link` does not work.

https://stackblitz.com/edit/github-7wtk2c?file=content/0.index.md

**Comment by XenBG** [comment]
I'm bringing up this topic because it is an extremely important and useful option that is available in many other documentation systems. I look forward to your solution as well in the form of grouped cards `<CardGrid />`.
